platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less

```
@@ -622,6 +622,11 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     ul li {
       list-style-type: disc;
     }
+    .video {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+    }
```

No need to use flex here. Use the same behaviour as we already have with the embedded object : 
```suggestion
    .video {
      width:320px;
      margin-left:auto;
      margin-right:auto;
    }
```